internal/provisioner/equality/equality_test.go

```diff
@@ -531,6 +531,13 @@ func TestNodePortServiceChanged(t *testing.T) {
 			},
 			expect: true,
 		},
+		{
+			description: "if the ports number changed",
```

Comment:
just a tiny nit on the description
```suggestion
			description: "if the number of ports changed",
```

Reply:
done